src/Components/MainComponent/Menu/index.js

```diff
@@ -0,0 +1,29 @@
+import FoodItem from "./FoodItem";
+
+const Menu = ({menuItems, showingMenuItems}) => {
+
+    if (menuItems.foodItems === undefined) {
+
+        return (
+            <div>
+            </div>
+        );
+    }
+
+    return (
+        <div className={showingMenuItems} >
+        <div className="d-flex flex-wrap justify-content-start">
```

Comment:
This div is inside the preceding one, hence needs indenting. This will affect the indenting of following lines.